src/exampleTypes/TagsText.js

```diff
@@ -34,26 +33,32 @@ let Text = ({
         onChange={e => tree.mutate(node.path, { operator: e.target.value })}
         options={operatorOptions}
       />
-      <TagsInput
-        splitCommas
-        tags={node.values}
-        onTagClick={tag => {
-          F.on(open)()
```

Comment:
After removing  the set open to true, I do not think the popover menu will show up